core-bundle/src/Security/DataContainer/CreateAction.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace Contao\CoreBundle\Security\DataContainer;
+
+class CreateAction extends AbstractAction
+{
+    use NewTrait;
+
+    public function __construct(
+        private string $dataSource,
```

Comment:
this should not be a property but only passed to the parent class I guess?

```suggestion
        string $dataSource,
```

Reply:
Ah right :) Done